calico_containers/calico_ctl/status.py

```diff
@@ -35,29 +36,34 @@ def status(arguments):
     this file's docstring with docopt
     :return: None
     """
-    calico_node_info = filter(lambda container: "/calico-node" in
-                              container["Names"],
-                              docker_client.containers())
-    if len(calico_node_info) == 0:
-        print "calico-node container not running"
-    else:
-        print "calico-node container is running. Status: %s" % \
-              calico_node_info[0]["Status"]
-
-        libraries_cmd = docker_client.exec_create("calico-node",
-                                                  ["sh", "-c",
-                                                   "cat libraries.txt"])
-        libraries_out = docker_client.exec_start(libraries_cmd)
-        result = re.search(r"^calico\s*\((.*)\)\s*$", libraries_out,
-                           re.MULTILINE)
-
-        if result is not None:
-            print "Running felix version %s" % result.group(1)
-
-        print "\nIPv4 BGP status"
-        pprint_bird_protocols(4)
-        print "IPv6 BGP status"
-        pprint_bird_protocols(6)
+    try:
+        calico_node_info = filter(lambda container: "/calico-node" in
+                                  container["Names"],
+                                  docker_client.containers())
+        if len(calico_node_info) == 0:
+            print "calico-node container not running"
+        else:
+            print "calico-node container is running. Status: %s" % \
+                  calico_node_info[0]["Status"]
+
+            libraries_cmd = docker_client.exec_create("calico-node",
+                                                      ["sh", "-c",
+                                                       "cat libraries.txt"])
+            libraries_out = docker_client.exec_start(libraries_cmd)
+            result = re.search(r"^calico\s*\((.*)\)\s*$", libraries_out,
+                               re.MULTILINE)
+
+            if result is not None:
+                print "Running felix version %s" % result.group(1)
+
+            print "\nIPv4 BGP status"
+            pprint_bird_protocols(4)
+            print "IPv6 BGP status"
+            pprint_bird_protocols(6)
+    except ConnectionError:
+        print "Docker is not running"
```

Comment:
While I do agree with these suggestions, I don't believe they fall within the scope of the issue this PR is trying to address. My goal in this PR is only to allow users to launch a dockerless node. Improvements to `checksystem` and `statuts` to provide useful information when docker has launched in this way are definitely the next step, but require significant thought into how calicoctl is structured. I think it'd be best if we start that work in a new PR after a bit of designing